### PR TITLE
Ear 1564 themed panel component

### PR DESCRIPTION
**Pull request by oms09**
### What is the context of this PR?

> - Panel Component/ Error Summary Component Redesign.

### How to review

> - Check the theme colour in the storybook matches the ons design system.
> - Check the prop of paragraphLabel matches the border colour.

### CODE REVIEW

**Review by zimboflyman — COMMENTED** [review]
All looks good except one small thing.
According to the design system - the error with header panel should have each error as a lin
Err panel Example
k?

**Comment by oms09** [comment]
> All looks good except one small thing. According to the design system - the error with header panel should have each error as a lin Err panel Example k?

File: eq-author/src/components-themed/panels/panel.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        <>
+          <Header>
+            <HeaderLabel> {headerLabel} </HeaderLabel>
+          </Header>
+        </>
+      )}
+      {variant === "success" && (
+        <SuccessPanelIconContainer>
+          <SpanIcon>
+            <StyledImage src={icon} />
```
Comment (martyncolmer) [inline eq-author/src/components-themed/panels/panel.js:154]: As the success icon doesn't change, we should hard code it rather than pass it as a prop.

**Review by martyncolmer — CHANGES_REQUESTED** [review]

File: eq-author/src/components-themed/panels/panel.js (hunk 1 above)
Comment (martyncolmer) [reply eq-author/src/components-themed/panels/panel.js:154]: Also don't need the img tag if we are just importing the svg

**Review by zimboflyman — DISMISSED** [review]
Nice!

File: eq-author/src/components-themed/panels/panel.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  variant: PropType.string,
+  children: PropType.node,
+  paragraphLabel: PropType.string,
+  headerLabel: PropType.string,
+  withLeftBorder: PropType.bool,
+};
+
+Panel.defaultProps = {
+  variant: "info",
+  type: "panel",
```
Comment (martyncolmer) [inline eq-author/src/components-themed/panels/panel.js:172]: Don't need to set the type

**Review by martyncolmer — CHANGES_REQUESTED** [review]

**Review by zimboflyman — APPROVED** [review]

**Review by martyncolmer — APPROVED** [review]

**Pull request merged**